test/activity-calendar.test.js

```diff
@@ -420,4 +420,62 @@ describe("ActivityCalendar", async function () {
       );
     });
   });
+
+  describe("week templates", async function () {
+    const createSymbol = async (name) => {
+      return activityCalendar.createSymbol(
+        "/images/1.jpg",
+        name,
+        "activities",
+        "25px",
+        "25px",
+        0,
+        popularCategory.id
+      );
+    };
+
```

Comment:
Copied this `createSymbol` from other tests. Just curious, if there's a reason we shouldn't consolidate this and similar definitions into one in the outer scope?